src/cache_cf.cc

```diff
@@ -4865,6 +4869,22 @@ static void free_ftp_epsv(acl_access **ftp_epsv)
     FtpEspvDeprecated = false;
 }
 
+static void parse_collapsed_forwarding_access(acl_access **collapsed_forwarding_access)
+{
+    aclParseAccessLine("collapsed_forwarding_access", LegacyParser, collapsed_forwarding_access);
+}
+
+static void free_collapsed_forwarding_access(acl_access **collapsed_forwarding_access)
+{
+    free_acl_access(collapsed_forwarding_access);
+}
+
+static void dump_collapsed_forwarding_access(StoreEntry *entry, const char *name, acl_access *collapsed_forwarding_access)
+{
+    if (collapsed_forwarding_access)
+        dump_SBufList(entry, collapsed_forwarding_access->treeDump(name, Acl::AllowOrDeny));
```

Comment:
Remove all three custom parse/free/dump functions as not needed -- you are reusing an existing `acl_access` type with no complications AFAICT.